configs/terraform/modules/artifact-registry/variables.tf

```
@@ -109,3 +109,24 @@ variable "cleanup_policies" {
   }))
   default = []
 }
+
+variable "remote_repository_config" {
+  type = object({
+    description = optional(string)
+    docker_repository = optional(object({
+      public_repository = string
+    }))
+    upstream_credentials = optional(object({
+      username_password_credentials = optional(object({
+        username                = string
+        password_secret_version = string
+      }))
+    }))
+  })
+  default = {
+    description = "Mirror of Docker Hub"
```

This is a description of remote repository which is not a mirror.
```suggestion
    description = "Docker Hub"
```

---

Removed default vars.